docs/deploy_narrative.md

```diff
@@ -50,7 +52,7 @@ To see your changes once the new image is created, simply open a narrative on CI
 ### Stage a Final Release
 
 1.  Update versions following (roughly) [semantic versioning](https://semver.org). The files to be updated are:
-    -   `package.json`, `package-lock.json`, `bower.json`, `src/config.json`, `src/config.json.templ`, `src/biokbase/narrative/\_\_init\_\_.py`
+    -   `package.json`, `package-lock.json`, `src/config.json`, `src/config.json.templ`, `src/biokbase/narrative/\_\_init\_\_.py`
```

Comment:
it looks like the last entry in that list is a copy-pasta from somewhere -- the `\_\_` render as written so it looks rather odd!

Reply:
Ha! I seem to recall an ancient version where the markdown renderer insisted that those should make it italicized. Guess that's updated. Fixed!